sci-misc/boinc/boinc-7.2.0-r1.ebuild

```diff
@@ -23,7 +23,7 @@ RDEPEND="
 	!sci-misc/boinc-bin
 	!app-admin/quickswitch
 	>=app-misc/ca-certificates-20080809
-	dev-libs/openssl
+	dev-libs/openssl:*
```

Comment:
A realy short look into the [configure.ac](https://github.com/BOINC/boinc/blob/master/configure.ac) showed that ssl and jpeg are checked at buildtime. Thus `:=` would be the right here and for jpeg?

I think I misunderstood `:*`. The [wiki](https://devmanual.gentoo.org/general-concepts/dependencies/) syas that `:* means that any slot is acceptable, and explicitly declares that changes in the slot or sub-slot can be ignored;`. I thought that means that changes within the sub-slot can be ignored, which would be okay. Anyway, also if the slot is changed the package depending on it isn't recompiled, correct? Further `:=` leads to recompiling of the package if openssl changes.
Please correct me if I am wrong again.